stable/prometheus-operator/templates/exporters/core-dns/servicemonitor.yaml

```diff
@@ -8,7 +8,7 @@ metadata:
     app: {{ template "prometheus-operator.name" . }}-coredns
 {{ include "prometheus-operator.labels" . | indent 4 }}
 spec:
-  jobLabel: jobLabel
+  jobLabel: {{ default "jobLabel" .Values.coreDns.serviceMonitor.jobLabel }}
```

Comment:
It is not necessary to specify a default value as well as a values file value - only the values file configuration is necessary and preferred: 
> The use of the default function should be avoided if possible in favor of defaults in values.yaml
https://github.com/helm/charts/blob/master/REVIEW_GUIDELINES.md#configuration